Formula/daemontools.rb

```diff
@@ -15,16 +16,20 @@ class Daemontools < Formula
 
   def install
     cd "daemontools-#{version}" do
+      inreplace ["package/run", "src/svscanboot.sh"] do |s|
+        s.gsub! "/service", "#{etc}/service"
+      end
+
       system "package/compile"
       bin.install Dir["command/*"]
+      mkdir_p etc/"service"
     end
   end
 
   def caveats
     <<~EOS
-      You must create the /service directory before starting svscan:
-        sudo mkdir /service
-        sudo brew services start daemontools
+      Create your service directories under:
```

Comment:
But brew already did. Could we just copy the old one as `post_install`?

Reply:
This refers to the individual service directories, which are created by the user below `service`.

Reply:
Good idea about copying existing services from `/service/*`. I’d prefer symlinks though.

Reply:
@SMillerDev Pushed a new version.